src/utils/image_utils.py

```diff
@@ -0,0 +1,90 @@
+"""
+Module: image_utils.py
+
+
+Contains functions that are used to analyze images.
+"""
+
+import itertools
+from typing import List, Optional
+
+from PIL import Image
+
+from src.common._io import ImageCropSelection
```

Comment:
Should this be an import from `src.structs.selections` instead?

```suggestion
from src.structs.selections import ImageCropSelection
```